packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -47,21 +71,89 @@ export class CoinbaseWalletProvider extends ProviderEventEmitter implements Prov
       if (!this.signer) {
         switch (args.method) {
           case 'eth_requestAccounts': {
-            const signerType = await this.requestSignerSelection(args);
+            // this causes a popup which we dont want.
+            //const signerType = await this.requestSignerSelection(args);
+            const signerType = 'scw';
```

Comment:
we still need the popup so users can select whether or not they want to use a scw or walletlink

Reply:
I may have just removed this in the moment because it was annoying